### PR TITLE
[AT-33] Transition sharpening poetry 

### PR DESCRIPTION
**Pull request by thaisbendixen**
The block-processing repository is being transitioned into poetry, hence all the submodules need to be transitioned as well.

- [x] Run make test, make build and make e2e
- [ ] Remove requirements.txt
- [x] Update README
- [ ] Update version (not required for now)
- [ ] ...

### CODE REVIEW

**Comment by chrieke** [comment]
As poetry noob using `poetry install` fails for me with `Running setup.py install for typed-ast did not run successfully`. Multiple times, new env. Not sure why, rest of the error message isn't very descriptive. `pip install typed-ast` works.

**Comment by thaisbendixen** [comment]
> As poetry noob using `poetry install` fails for me with `Running setup.py install for typed-ast did not run successfully`. Multiple times, new env. Not sure why, rest of the error message isn't very descriptive. `pip install typed-ast` works.

Ugh.. this is a red flag. Lets chat abou this today if you have time.

File: README.md [hunk 1]
```diff
@@ -43,23 +43,35 @@ cd sharpening
...
-mkvirtualenv --python=$(which python3.7) up42-sharpening
+pyenv version
 ```
 
-Activate the new environment:
+To activate the poetry environment, run:
 
 ```bash
-workon up42-sharpening
+poetry shell
```
Comment (jmigueldelgado) [inline README.md:67]: For some reason (probably a previous pyenv python version installed) at this point I got:
```
The currently activated Python version 3.10.2 is not supported by the project (>=3.8,<3.10).
Trying to find and use a compatible version. 

  NoCompatiblePythonVersionFound

  Poetry was unable to find a compatible version. If you have one, you can explicitly use it via the "env use" command.
```
and had to run `poetry env use /Users/jose.delgado/.pyenv/versions/3.8.8/bin/python3` to continue.

Something like `poetry env use $HOME/.pyenv/versions/3.8.8/bin/python3` could be added to the README at this point.

**Review by jmigueldelgado — DISMISSED** [review]
Except for the comment that I added to README.md it works like a charm. Great job!

File: README.md (hunk 1 above)
Comment (bkavlak) [reply README.md:67]: it was the same for me

**Review by seedlit — COMMENTED** [review]
Works for me as well.

**Pull request closed without merging**